patches/crop_decoder.patch

```diff
@@ -0,0 +1,163 @@
+diff --git a/jdcolor.c b/jdcolor.c
+index 8da2b4ea..c5b82279 100644
+--- a/jdcolor.c
++++ b/jdcolor.c
+@@ -516,6 +516,35 @@ rgb_rgb_convert(j_decompress_ptr cinfo, JSAMPIMAGE input_buf,
```

Comment:
Can we split the patches into 2 parts?
1. cmyk format
2. crop decode

the crop decode can be handed in your code, it's not below libjpeg, right? @Liangliang-Ma 

also, please use git format to generate patches.

Reply:
Done. Sorry for late response.